config.json

```diff
@@ -3,13 +3,14 @@
   "checklist_issue": 2,
   "exercises": [
     {
-      "core": false,
-      "difficulty": 1,
-      "slug": "hello-world",
-      "topics": null,
-      "unlocked_by": null,
-      "uuid": "3596bbf7-66f0-49c0-8c3e-3472455f90c7"
-    },
+     "uuid": "3596bbf7-66f0-49c0-8c3e-3472455f90c7",
+     "slug": "hello-world",
+     "core": false,
+     "unlocked_by": null,
+     "difficulty": 1,
+     "topics": [
+
+    ]
```

Comment:
The missing coma `,` makes the JSON invalid.

Reply:
Oh, the real problem is not a missing comma as I realise now, its the missing closing curly brace (`{`)…